### DISCUSSION TITLE
Get more violated tags for the same struct field

### DISCUSSION THREAD

**Discussion by oSethoum** [go-playground/validator#1127] [Q&A]
I have this example: 
```go
type User struct {
	Email string `validate:"min=10,email"`
}

func main() {
	validate := validator.New()

	err := validate.Struct(&User{Email: "coder"})
	if err != nil {
		println(err.Error())
	}
}

```
clearly the Email field here is violating two tags `min` and `email` but I only get one tag at a time not both in the same time is there a way to fix this? 
this is the error i get 
`
Key: 'User.Email' Error:Field validation for 'Email' failed on the 'min' tag`

**Comment by deankarn** [comment]
NO validator is not designed to support running and returning all validations, it returns on the first failed validation encountered; however this is one reason that `alias` exists to create a custom tag encompassing one or more individual validations.

**Reply by mhmmdnaufall** [reply]
Is there any way to overcome this? Any tips?
